src/components/constants.tsx

```diff
@@ -0,0 +1,7 @@
+import { QoS } from "mqtt";
+
+export const QoSOptions: { [key in QoS]: string } = {
+  0: "0 - At most once",
+  1: "1 - At least once",
+  2: "2 - Exactly once",
+} as const;
```

Comment:
@sergei-malyshko @h2xd 

After our discussion I had a second thought:

That^ might be a better way to define a single point of QoS options. This can be used not only to populate the options of `select` elements but also to display more descriptive labels in the tables or anywhere that a QoS value needs to be presented in the user interface.